webapp-django/crashstats/signature/tests/test_views.py

```diff
@@ -13,7 +15,24 @@
 DUMB_SIGNATURE = 'hang | mozilla::wow::such_signature(smth*)'
 
 
+@contextlib.contextmanager
+def gvfp_mock(value=None):
+    """Mock get_versions_for_product() so it doesn't hit supersearch"""
+    value = value if value is not None else ['20.0', '19.1', '19.0', '18.0']
+    with mock.patch('crashstats.crashstats.utils.get_versions_for_product') as mock_gvfp:
+        mock_gvfp.return_value = value
+        yield
+
+
 class TestViews(BaseTestViews):
+    def setUp(self):
+        super(TestViews, self).setUp()
+        self._gvfp_patch = mock.patch('crashstats.crashstats.utils.get_versions_for_product')
+        mocked_gvfp = self._gvfp_patch.start()
+        mocked_gvfp.get.side_effect = mocked_gvfp
+
+    def tearDown(self):
+        super(TestViews, self).tearDown()
```

Comment:
Oops--don't need this bit anymore.

Reply:
And by "don't need this bit", I mean "there's two levels of mocking and one of them is unneeded".

I wanted to do this as a pytest fixture, but it requires us to un-unittest everything. Blech.